### PR TITLE
3rd function

### PR DESCRIPTION
**Pull request by codeit13**
### Pull Request
Before creating a PR, just answer a below questions to get started...

1. What type of change does your PR bring?
    - [     ] Bug fix
    - [✔️ ] Feature Update
    - [     ] Issue fix
    - [     ] Doc fix/update
    - [     ] Other

2. What OS you're using right now?
    - [     ] Windows 7 & above
    - [ ✔️] Linux (Ubuntu, Debian)
    - [     ] macOS
  
3. If this PR fixes a bug, refer to the issue number 

    Solves Issue: #21 
  
4. If this PR brings a feature, what kind it is?
` Add 3rd function to implement files without TXT files`
  
5. If this PR is related to an issue or it's discussed, refer issue here
    
    Issue: #21 
 
6. Any else you want to convey?
     
    No

7. Does your PR work in a stable manner?
    - [✔️ ] Yes
    - [      ] No

### CODE REVIEW

**Comment by codeit13** [comment]
@alphaX86 I have made Issue #21 in this PR Complete.
Here Greetings Check is not passed I don't know why Can you pls look into it.

**Comment by alphaX86** [comment]
Hmm, seems I need to refine that CI a bit. Leave that work flow and the GitLab one. I'll check the functions and return to you

File: codeit/codeit.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import os
 from clint.textui import colored
 from codeit.codeitHelp import help
-from codeit.codeitInit import init, init_noerror, init_noerror_agg
+from codeit.codeitInit import init, init_noerror, init_noerror_agg, create_only_files
```
Comment (alphaX86) [inline codeit/codeit.py:6]: Function name is good!

File: codeit/codeit.py [hunk 2]
```diff
@@ -48,16 +48,19 @@ def main():
                     contestName = sys.argv[countArg]
                     fileNames = get_filename()
                     init(contestName, fileNames)
+            elif arg == "-f":
+                contestName = sys.argv[countArg]
+                fileNames = get_filename()
+                create_only_files(contestName, fileNames)
```
Comment (alphaX86) [inline codeit/codeit.py:54]: Tag seems pretty nice!

File: codeit/codeitHelp.py [hunk 3]
```diff
@@ -17,8 +17,9 @@ def help():
     print(colored.red('Args list\n'))
     print(colored.yellow("-h -------------------------- " + colored.blue("To display help menu")))
     print(colored.yellow("-i <contest> ---------------- " + colored.blue("To initialize a contest directory")))
+    print(colored.yellow("-f <contest> ---------------- " + colored.blue("To create only files in the current directory level")))
```
Comment (alphaX86) [inline codeit/codeitHelp.py:20]: Hmm, this looks neat to me! Now, for system test! ;)

File: codeit/codeitInit.py [hunk 4]
```diff
@@ -27,11 +27,23 @@ def init(contestName,fileNames):
...
 
+def create_only_files(contestName,fileNames):
+  # create a directory with contest name
+  try:
+    # create files for contest (should be 6 cpp files)
+    currentDir = os.getcwd()
+    for files in range(len(fileNames)):
+      write_to_file(fileNames[files], template_cp(), currentDir)
+  except OSError as e:
+    print(colored.red("Failed! The Files can't be created in current Directory."))
```
Comment (alphaX86) [inline codeit/codeitInit.py:38]: Ok, seems you made it like a within folder type and not separate one

**Review by alphaX86 — CHANGES_REQUESTED** [review]
I need this change alone

**Comment by alphaX86** [comment]
Do that change alone and I'll merge, consider it as approved @codeit13

File: codeit/codeitHelp.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (codeit13) [reply codeit/codeitHelp.py:20]: You mean like this `To create only files in the current directory level folder needed`

File: codeit/codeitHelp.py [hunk 5]
```diff
@@ -17,8 +17,9 @@ def help():
     print(colored.red('Args list\n'))
     print(colored.yellow("-h -------------------------- " + colored.blue("To display help menu")))
     print(colored.yellow("-i <contest> ---------------- " + colored.blue("To initialize a contest directory")))
+    print(colored.yellow("-f <contest> ---------------- " + colored.blue("To create only files in the current directory level folder needed")))
```
Comment (alphaX86) [inline codeit/codeitHelp.py:20]: This is what I expected... Anyways, great job 👍

**Review by alphaX86 — APPROVED** [review]
Approved! 👍

File: codeit/codeitHelp.py (hunk 5 above)
Comment (codeit13) [reply codeit/codeitHelp.py:20]: Yups!

**Pull request merged**